diff --git a/go.mod b/go.mod
index 431a56a..a2455ec 100644
--- a/go.mod
+++ b/go.mod
@@ -2,6 +2,11 @@ module github.com/NordSecurity/nordvpn-linux
 
 go 1.20
 
+replace (
+	moose/events => ./third-party/moose-events/moosenordvpnappgo
+	moose/worker => ./third-party/moose-worker/mooseworkergo
+)
+
 require (
 	github.com/NordSecurity/gopenvpn v0.0.0-20230117114932-2252c52984b4
 	github.com/NordSecurity/libdrop v1.1.2-0.20240214140349-bc7d2cffb948
@@ -39,6 +44,8 @@ require (
 	google.golang.org/grpc v1.52.0-dev
 	google.golang.org/protobuf v1.30.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
+	moose/events v0.0.0
+	moose/worker v0.0.0
 )
 
 require (
