diff --git a/go.mod b/go.mod
index 2446a1e4..419164db 100644
--- a/go.mod
+++ b/go.mod
@@ -4,6 +4,11 @@ go 1.21.1
 
 toolchain go1.22.2
 
+replace (
+	moose/events => ./third-party/moose-events/moosenordvpnappgo
+	moose/worker => ./third-party/moose-worker/mooseworkergo
+)
+
 require (
 	github.com/NordSecurity/gopenvpn v0.0.0-20230117114932-2252c52984b4
 	github.com/NordSecurity/libdrop v1.1.2-0.20240214140349-bc7d2cffb948
@@ -44,6 +49,8 @@ require (
 	google.golang.org/grpc v1.56.3
 	google.golang.org/protobuf v1.33.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
+	moose/events v0.0.0
+	moose/worker v0.0.0
 )
 
 require (
