diff --git a/go.mod b/go.mod
index 25f066ce..fee14c21 100644
--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,9 @@ module github.com/NordSecurity/nordvpn-linux
 
 go 1.20
 
+replace (
+	moose/events => ./third-party/moose-events/moosenordvpnappgo
+	moose/worker => ./third-party/moose-worker/mooseworkergo
+)
+
 require (
@@ -37,6 +42,8 @@ require (
 	google.golang.org/grpc v1.56.3
 	google.golang.org/protobuf v1.30.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
+	moose/events v0.0.0
+	moose/worker v0.0.0
 )
 
 require (
