diff --git a/go.mod b/go.mod
index e1e88ce..9b6608d 100644
--- a/go.mod
+++ b/go.mod
@@ -4,12 +4,19 @@ go 1.21.1
 
 toolchain go1.22.2
 
+replace (
+	moose/events => ./third-party/moose-events/moosenordvpnappgo/v14
+	moose/worker => ./third-party/moose-worker/mooseworkergo/v14
+)
+
 // Bindings
 // NOTE: If you are chaning the binding versions here, keep in mind that you
 // may also need to update versions in `./lib-versions.env` file.
 require (
 	github.com/NordSecurity/libdrop-go/v8 v8.0.0-20241017064027-670787595588
 	github.com/NordSecurity/libtelio-go/v5 v5.0.3
+	moose/events v0.0.0
+	moose/worker v0.0.0
 )
 
 require (
